test/unit/Sensorario/DiamondKata/DiamondKataTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace Sensorario\DiamondKata;
+
+use PHPUnit_Framework_TestCase;
+
+final class DiamondKataTest extends PHPUnit_Framework_TestCase
+{
+    public function testOneDimentionalMatrix()
+    {
+        $matrix = DiamondKata::createFrom('A');
```

Comment:
A DiamondKata class ... who create a "matrix" .... but in true is a DiamondKata instance .... we perhaps need two separated classes?